src/util.js

```diff
@@ -49,14 +48,7 @@ function replaceCIDbyTAG (dagNode) {
 
     const keys = Object.keys(obj)
 
-    if (keys.length === 1 && keys[0] === '/') {
```

Comment:
can you explain this removal? is this a v0 vs v1 thing?

Reply:
CIDs used to be encoded as JSON objects `{"/": "base-encodec-cid"}`.
These days CIDs are encoded as CID objects. That code was there to also allow those old style CIDs. But that would the encoder fail if you have e.g. `{"/": 8}`. It closes https://github.com/ipld/js-ipld-dag-cbor/issues/84.